_overviews/tutorials/binary-compatibility-for-library-authors.md

```diff
@@ -0,0 +1,140 @@
+---
+layout: singlepage-overview
+title: Binary Compatibility for library authors
+
+discourse: true
+permalink: /tutorials/:title.html
+---
+
+## Introduction
+
+A diverse and comprehensive set of libraries is important to any productive software ecosystem. While it is easy to develop and distribute Scala libraries, good library authorship goes far
+beyond just writing code and publishing them.
+
+In this guide, we will cover the important topic of **Binary Compatibility**:
+
+* How binary incompatibility can cause production failures in your applications
+* How library authors can avoid breaking binary compatibility, and/or convey breakages clearly to library users when they happen
```

Comment:
I would split this last item in two:
* How library authors can avoid breakages in binary compatibility
* How to reason about the impact of binary incompatible changes

Reply:
Are there different levels of "impact` for binary incompatibility?

Reply:
Not really. In hindsight, that last bullet point doesn't explain what I wanted to. Let me reword it: "How to reason about the impact of changes in their code". And I would put it in the second position instead of the last one.